src/backend/distributed/executor/multi_real_time_executor.c

```diff
@@ -83,6 +83,9 @@ MultiRealTimeExecute(Job *job)
 	bool allTasksCompleted = false;
 	bool taskCompleted = false;
 	bool taskFailed = false;
+	bool sizeLimitIsExceeded = false;
+	DistributedExecutionStats *executionStats = palloc0(
+		sizeof(DistributedExecutionStats));
```

Comment:
You could also consider allocating this on the stack:

```c
DistributedExecutionStats executionStats = { 0 };
```
and then passing it by pointer to `ManageTaskExecution(..., &executionStats)`